CONTRIBUTING.md

```diff
@@ -106,7 +106,7 @@ _**All commit, that not applies to these rules, should be split by these rules.
  - Update changelog;
  - Update documentation;
  - Cover new functionality by tests;
- - Your code should be written according to a codestyle like in sc-machine.
+ - Your code should be written according to a codestyle like in scp-machine.
```

Comment:
where can I find that codestyle?

Reply:
Then maybe it's best to leave the sc-machine here for now.

Reply:
https://ostis-ai.github.io/sc-machine/dev/codestyle/